test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -142,16 +142,28 @@ bootstrap_remote_freebsd()
 
     if [ "${controller}" ]; then
         jinja2_pkg="py${python_package_version}-jinja2"
-        cryptography_pkg="py${python_package_version}-cryptography"
         pyyaml_pkg="py${python_package_version}-yaml"
+        cryptography_pkg="py${python_package_version}-cryptography"
+		packaging_pkg="py${python_package_version}-packaging"
 
         # Declare platform/python version combinations which do not have supporting OS packages available.
         # For these combinations ansible-test will use pip to install the requirements instead.
         case "${platform_version}/${python_version}" in
+            14.0/3.9)
+				# all available
+				packages="
+					$packages
+					${packaging_pkg}
+				"
+				;;
             *)
                 jinja2_pkg=""  # not available
                 cryptography_pkg=""  # not available
                 pyyaml_pkg=""  # not available
+				packages="
+					$packages
+					rust
+				"
```

Comment:
Update the wheel matrix (https://github.com/ansible/spare-tire/blob/main/wheel_matrix.yml) so we have wheels available. That will avoid the need to install rust here.

Reply:
i did, will remove rust then